File: library/src/main/java/com/androidwind/androidquick/module/glide/support/ImageListener.java
package com.androidwind.androidquick.module.glide.support;

/**
 * @author  ddnosh
 * @website http://blog.csdn.net/ddnosh
 */
public interface ImageListener<T> {

    void onSuccess(T result);

    void onFail(Throwable throwable);
}

File: sdk/src/main/java/com/github/ddnosh/arabbit/module/glide/support/ImageListener.kt
package com.github.ddnosh.arabbit.module.glide.support

/**
 * @author ddnosh
 * @website http://blog.csdn.net/ddnosh
 */
interface ImageListener<T> {

    fun onSuccess(result: T)

    fun onFail(throwable: Throwable)
}
